### DISCUSSION TITLE
Right align keys

### DISCUSSION THREAD

**Discussion by leana8959** [grtcdr/macchina#273] [Q&A]
Hello,

I'd like to know if it's possible to align the keys against the separators?  
Something like this:
```
       Host / leana@mbp.wired.lan
    Machine / MacBookPro18,2
     Kernel / 22.3.0
```

Thanks! I appreciate the project <3

**Comment by grtcdr** [comment]
I don't think this is currently possible.

**Reply by leana8959** [reply]
That's unfortunate, do you mind sharing some of the technical details? I'm still a newbie in Rust.  
I look forward to the day when I can contribute to the project!

**Reply by grtcdr** [reply]
> I look forward to the day when I can contribute to the project!

That's really great! However, I really haven't messed about with macchina in a long time, so I don't remember many of our UI implementation details.
